src/components/sprite-info/sprite-info.css

```diff
@@ -31,7 +31,7 @@
 
 .icon-wrapper {
     display: inline-block;
-    box-sizing: content-box;
+    box-sizing: content-box !important;
```

Comment:
Same here about `!important`.  I think it might be easier to just adjust the widths and heights so that we don't actually need to use `box-sizing: content-box` at all.